CK.ActivityMonitor/Client/ActivityMonitorTextWriterClient.cs

```diff
@@ -140,7 +140,7 @@ protected override void OnGroupOpen( IActivityLogGroup g )
             var w = _buffer.Clear();
             string levelLabel = g.MaskedGroupLevel.ToString();
             string start = string.Format( "{0}> {1}: ", _prefix, levelLabel );
-            _prefix += "|  ";
+            _prefix += "\u2502  ";
```

Comment:
Why not use _emptyLinePrefix defined at https://github.com/Invenietis/CK-ActivityMonitor/pull/2/files#diff-b4be1a882deaf0db7804645f38cf275aR43, to avoid hardcoding?

Reply:
ActivityMonitorTextWriterClient may actually support options for display... 
But... I'm wondering @SuperFola: do you have a real use case for this... Seems quite unuseful to me.

@Kuinox, @bcrosnier: any thoughts?

Reply:
A real use case to be able to change the _prefix? hum not really, but people love customization nowadays

Reply:
Well, the separator character could certainly use a `const` or otherwise common compile-time symbol if we want to eg. use its beefy big brother the [BOX DRAWINGS DOUBLE VERTICAL (U+2551)](https://www.fileformat.info/info/unicode/char/2551/index.htm).

As for making it customizable, I'm not technically against, but I'd be interested to seeing how it performs when using a runtime symbol instead of a compile-time symbol. A completely customizable `ActivityMonitorTextWriterClient` (and relevant `Helper`) could have its own separate implementation.

Reply:
+1 for the "A completely customizable ActivityMonitorTextWriterClient (and relevant Helper) could have its own separate implementation.". Definitely!